tests/Unit/PointwiseFunctions/EquationsOfState/Test_Tabulated.cpp

```diff
@@ -0,0 +1,135 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <string>
+#include <vector>
+
+#include "DataStructures/Matrix.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/Header.hpp"  // IWYU pragma: keep
+#include "IO/H5/ReadTabulated.hpp"
+#include "Informer/InfoFromBuild.hpp"
+#include "tests/Unit/TestingFramework.hpp"
+
+namespace {
+
+void test_tabulated() noexcept {
+  h5::H5File<h5::AccessType::ReadOnly> sample_file(unit_test_path() +
+                                                   "/IO/KDE_Sample.h5");
+  const auto& sample_data = sample_file.get<h5::Tabulated>("/");
```

Comment:
You should also test a subgroup in the HDF5, e.g. `/sample_data` which contains all the same data as the root.